README.md

```diff
@@ -26,57 +30,35 @@ Python 3.6
 
 ## Quickstart
 
-### Important commands
-
-```
-cld --help # lists available commands
-cld search --help 	# Search API usage
-cld admin     # Admin API functions
-cld uploader    # Upload API functions
-```
-
-Using temporary Cloudinary configurations requires the `-c` option or `--config`:
-
-```
-cld -c <CLOUDINARY_URL> <COMMAND> <OPTIONS> <PARAMS>
-```
-
-Additional configurations can be used by using the `-C` option.
-
 ```
-cld -C my_subaccount admin resources
+Usage: cld [OPTIONS] COMMAND [ARGS]...
 ```
 
-## Additional configurations
+### Important commands
 
 ```
-Usage: cld config [OPTIONS]
-
-  Display current configuration
-
-Options:
-  -n, --new TEXT...  Set an additional configuration
-                     eg. cld config -n <NAME> <CLOUDINARY_URL>
-  -ls, --ls          List all configurations
-  -rm, --rm TEXT     Delete an additional configuration
-  -url, --from_url TEXT  Create a configuration from a Cloudinary URL
-  --help             Show this message and exit.
+cld --help         # Lists available commands
+cld search --help  # Shows usage for the Search API
+cld admin          # Lists Admin API methods
+cld uploader       # Lists Upload API methods
 ```
 
 ## Upload API
 
 Bindings for the Upload API.
 
+You can find documentation for each of the Upload API methods at [https://cloudinary.com/documentation/image_upload_api_reference](https://cloudinary.com/documentation/image_upload_api_reference) 
+
 The basic syntax using the Upload API is as follows:
 
 ```
 Usage: cld uploader [OPTIONS] [PARAMS]...
```

Comment:
Would add here the --help option

Reply:
Yes, I think it's a good starting point for someone who doesn't want to read the documentation